src/server.ts

```diff
@@ -17,12 +20,13 @@ app.use(bodyParser.urlencoded({ extended: true }))
 app.use(cors())
 RegisterRoutes(app)
 app.use(exceptionHandler)
+app.use('/swagger.json', express.static(__dirname + '/swagger/swagger.json'))
+app.use('/', swaggerUI.serve, swaggerUI.setup(swaggerJSON))
```

Comment:
Napisałem w tasku, pod jakim adresem chcielibyśmy hostować naszego swaggera, hostowanie go pod głównym endpointem servera to mocno średni pomysł ;) Plus nie działa  u mnie hostowanie statyczne pliku `swagger.json` ;/